src/main/java/com/twingly/search/Query.java

```diff
@@ -0,0 +1,157 @@
+package com.twingly.search;
+
+import com.twingly.search.exception.TwinglyException;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * The type Query.
+ */
+public class Query {
+    private static final String BASE_URL = "http://api.twingly.com";
```

Comment:
All right, no worries, reply with the info when you get a chance. If you could also add an example program using the client, I can help debugging it.